src/_layouts/docs.html

```diff
@@ -51,22 +51,26 @@ <h3 class="sidebar-sectionTitle">{{ section.title | escape }}</h3>
 
   </nav>
   <article class="docsContent">
-    {% if page.hideEditButton != true %}
-    <a class="editOnGitHub u-hideOnMobile" 
-       href="https://github.com/tilt-dev/tilt.build/tree/master/docs/{{page.name}}">
-      {% svg assets/svg/github.svg width=16 height=16 %}
-
-      <span class="editOnGitHub-text">
-        Edit on GitHub
-      </span>
-    </a>
-    {% endif %}
-    
     <h1>{{ page.title | escape }}</h1>
     
     {{ content }}
 
+    <div class="docsFooter">
     {% include helpful_form.html %}
+    <a id="scrollToTop" href="#">
+      <span aria-hidden="true">↑</span>
```

Comment:
(tiny nitpick) because this is an ascii character and accessible to assistive tech, i think you could leave it without any additional markup, but totally up to you! 

not applicable here, but as a sidenote, i like to use `role="presentation"` for icons and elements like this. the difference bw it and `aria-hidden` is that it says to assistive tech "this element is here, but it doesn't have semantic value to you."